.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,55 @@
+name: Cherry-pick a release
+
+on:
+  workflow_dispatch:
+    inputs:
+      amp-version:
+        description: AMP version (13 digits)
+        required: true
+        type: string
+      sha-1:
+        description: commit sha to cherry-pick with
+        required: true
+        type: string
+      sha-2:
+        description: (optional) commit sha to cherry-pick with
+        required: false
+        type: string
+#      TODO(estherkim): tie to promote workflow
+#       channel:
+#         description: channels to promote to
+#         required: true
+#         type: choice
```

Comment:
Do you know if this is a single or multiple choice?

Reply:
Single. Still thinking this through, if we want to auto promote or have on-duty do it manually.

Reply:
Yes, we want to recreate the existing workflows as close as possible. That includes automatically promoting CPs

Reply:
That's fine, this can be a string.

Reply:
Hmm... we can probably make lives easier here by just having a single "auto-promote affected channels" checkbox, then read the JSON file and for every AMP version that shares the first 10 digits just update that. It wouldn't be a big departure from how we operate now... wdyt?

Reply:
Sure, that's a good idea

Reply:
Alright, let's just replace this commented out section and add a comment about what we plan to do here in followup PRs. Approving to unblock for now!